src/picterra/__main__.py

```diff
@@ -19,8 +19,19 @@
 
 
 def parse_args(args):
+    # Instantiate client, just for checking URl and api key
+    try:
+        client = APIClient()
+        target = client.base_url
+        suffix = "Your API key starts with '%s'" % client.api_key[:4]
+        desc = "You are targeting %s.\n\n%s" % (target, suffix)
+        del client
```

Comment:
I'd put this code into a `check_api_key` helper function, this removes the need for the `del`